src/AdvancedMH.jl

```diff
@@ -51,14 +51,14 @@ logdensity(model::DensityModel, t::Transition) = t.lp
 
 # A basic chains constructor that works with the Transition struct we defined.
 function AbstractMCMC.bundle_samples(
-    ts::Vector{<:Transition},
+    ts::Vector{Transition{T, L}},
     model::DensityModel,
     sampler::MHSampler,
     state,
     chain_type::Type{Vector{NamedTuple}};
     param_names=missing,
     kwargs...
-)
+) where {T, L}
```

Comment:
```suggestion
)
```

Reply:
See above comment for why I can't do this one.